src/components/MigrationAlert/__tests__/MigrationAlert.test.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MigrationAlert } from '../index';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      query: { platform: 'react' }
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+
+describe('MigrationAlert', () => {
+  const legacyPageLinkText = 'View latest documentation',
+    currentPageLinkText = 'Looking for legacy docs?',
+    props = {
+      isLegacy: true,
+      url: '/[platform]/tools/cli/graphqlapi'
+    },
+    component = <MigrationAlert isLegacy={props.isLegacy} url={props.url} />;
+
+  it('should render the MigrationAlert component', async () => {
+    render(component);
+
+    const migrationAlertNode = props.isLegacy
```

Comment:
Instead of ternaries, we should split these scenarios into their own tests, like 'should render legacy text when isLegacy is true' so we can make the test a little more assertive since we have control of what we're rendering in the test. That way it will also be easier to track failures when they happen.